src/styles/blocks/card.scss

```diff
@@ -0,0 +1,80 @@
+.card {
+  box-sizing: border-box;
+  width: $card-width;
+  border: 1px solid $c-borderColor;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  color: $c-basicHeader;
+  font-size: $font-basic-size;
+  line-height: 14px;
+
+  &__image {
+    display: flex;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__tittle {
+    text-align: left;
+    margin-bottom: 4px;
+    line-height: 18px;
+    font-size: $font-basic-size;
+    font-weight: $roboto-medium;
+  }
+
+  &__review,
+  &__code {
+    font-size: 10px;
+  }
+
+  &__code,
+  &__price {
+    color: $c-specificGrey;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+  }
+
+  &__price-value {
+    font-size: 16px;
+    font-weight: $roboto-bold;
```

Comment:
how does font-weight relate to Roboto font?
as for me, u can name it more abstractive

```SCSS
$font-weight_regular: 400;
...
$font-weight_bold: 700;
...
```